libs/sql-schema-describer/src/postgres.rs

```diff
@@ -936,14 +979,17 @@ impl<'a> SqlSchemaDescriber<'a> {
                 ),
             });
 
-            let algorithm = match row.get_string("index_algo").as_deref() {
-                Some("btree") => Some(SqlIndexAlgorithm::BTree),
-                Some("hash") => Some(SqlIndexAlgorithm::Hash),
-                Some("gist") => Some(SqlIndexAlgorithm::Gist),
-                Some("gin") => Some(SqlIndexAlgorithm::Gin),
-                Some("spgist") => Some(SqlIndexAlgorithm::SpGist),
-                Some("brin") => Some(SqlIndexAlgorithm::Brin),
-                _ => None,
+            let algorithm = match row.get_expect_string("index_algo").as_str() {
```

Comment:
This might not be there. At least the ops will be null on CDB, check that you get a value for the algo and it doesn't crash...